packages/zowe-explorer-api/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@zowe/zowe-explorer-api",
+  "version": "1.10.0",
+  "description": "Extensibility API for Zowe Explorer.",
+  "publisher": "Zowe",
+  "author": "Zowe",
+  "license": "EPL-2.0",
+  "repository": {
+    "url": "https://github.com/zowe/vscode-extension-for-zowe"
+  },
+  "main": "lib/index.js",
+  "files": [
+    "lib",
+    "src"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn license && tsc -p ./",
```

Comment:
should `yarn` be in `devDependencies` so this runs a common version of `yarn` in `node_modules`?